src/setup-beam.js

```diff
@@ -321,6 +346,10 @@ function isStrictVersion() {
   return getInput('version-type', false) === 'strict'
 }
 
+function isExactVersion(spec) {
```

Comment:
I'd call this `isSemver`.
And below I believe you can remove the extra `&& true`.

Reply:
Right. I was thinking more like

```
return semver.parse(spec) ? true : false
```

(or an `if`) but yours works too, in that case.

It's just that since JavaScript would consider that semver to be a truthy value, the way we're using it in this pull request would still hold, unless it returned `0`, `""`, or `[]`.

It wouldn't if you were doing `isExactVersion(rebar3Spec) == true`.

The joys of JavaScript.

Your solution, though, is more semantically correct.

Reply:
I'd still rename the function for extra clarity, too.